kolibri_instant_schools_plugin/assets/src/views/sign-in-page/index.vue

```diff
@@ -135,6 +163,7 @@
       usernameBlurred: false,
       passwordBlurred: false,
       formSubmitted: false,
+      passwordResetModalVisble: false, // are we giving this a route?
```

Comment:
was thinking about giving this a route and giving it some state in vuex - left it at this for now. Thoughts?

Reply:
> was thinking about giving this a route

That doesn't sound right to me. Modals generally don't have routes associated with them, because you don't normally link to an open modal or close/open it by clicking 'back'/'forward'.

Reply:
ah alright. I'll leave it be, then